OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.kt

```diff
@@ -298,7 +304,7 @@ class MyCoursesListFragment : OfflineSupportBaseFragment(), RefreshListener {
                 }
             }
             IAPFlowData.IAPAction.PURCHASE_FLOW_COMPLETE -> {
-                onRefresh()
+                EventBus.getDefault().post(MyCoursesRefreshEvent())
```

Comment:
we can simply call `courseViewModel.fetchEnrolledCourses(type = CoursesRequestType.LIVE)` here

Reply:
replaced.